library/alloc/src/raw_vec.rs

```diff
@@ -259,6 +259,11 @@ impl<T, A: Allocator> RawVec<T, A> {
         Self { ptr: unsafe { Unique::new_unchecked(ptr) }, cap, alloc }
     }
 
+    pub(crate) unsafe fn from_nonnull_in(ptr: NonNull<T>, capacity: usize, alloc: A) -> Self {
```

Comment:
It's an unsafe method, so it should at least point to its sibling regarding the Safety section.